### PR TITLE
Revert new modes StopPlace_ abstract on stopPlacesInFrame_RelStructure

### LINKED ISSUE #512: Too generic type for stop places in SiteFrame
The pull request introducing new modes (https://github.com/NeTEx-CEN/NeTEx/commit/147c902f0df0eee1e727acdba9eed392581d0170)
modified the type of stop places in a site frame from` StopPlace `to the more generic `StopPlace_` :

https://github.com/NeTEx-CEN/NeTEx/blob/3d1d9e716ffd2577a58b0a7b6bf9770ed5bf3276/xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd#L113

where:
`<xsd:element ref="StopPlace" maxOccurs="unbounded"/>`
became:
`<xsd:element ref="StopPlace_" maxOccurs="unbounded"/>`

As a result, type checking is relaxed and it is now possible to refer to other types in SiteFrame/stopPlaces, such as:
```

<SiteFrame>
    <stopPlaces>
        <Parking/>
        <PointOfInterest/>
    </stopPlaces>
</SiteFrame>
```

These types have their own containers (`<parkings>,` `<pointOfInterests>`, ...) and should not be mixed with `<stopPlaces>`

We believe that this change was not intended and should be reverted.

### PR DESCRIPTION
**Pull request by skinkie**
Fix #512

### CODE REVIEW

**Comment by ue71603** [comment]
@skinkie  seems not to work.

**Comment by ue71603** [comment]
@skinkie   it needs to be the StopPlace_ otherwise the substution e.g. for TaxiRank is not working.
image

So I would not do this one. Or does it cause problems elsewhere?

**Comment by ue71603** [comment]
Currently under stopPlaces 
StopPlace and TaxiRank are possible. However, it would be more aligned with the rest to have
taxiRanks as a separate grouping within SiteFrame. Shall we adapt it.

**Comment by ue71603** [comment]
@Aurige  @nick-knowles 
Had to change this. stopPlaces now can no longer contain TaxiRank (and I think this is good, because we did this also for all other things. VehicleMeetingPoint never was allowed in stopPlaces so I changed the description.

**Review by ue71603 — APPROVED** [review]

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd [hunk 1]
```diff
@@ -105,14 +105,14 @@ Rail transport, Roads and Road transport
...
+			<xsd:documentation>Type for containment in frame of STOP PLACE</xsd:documentation>
 		</xsd:annotation>
 		<xsd:complexContent>
 			<xsd:extension base="containmentAggregationStructure">
 				<xsd:sequence>
-					<xsd:element ref="StopPlace_" maxOccurs="unbounded">
+					<xsd:element ref="StopPlace" maxOccurs="unbounded">
 						<xsd:annotation>
-							<xsd:documentation>Can be a STOP PLACE, VEHICLE MEETING POINT, TAXI RANK.</xsd:documentation>
+							<xsd:documentation>Can be a STOP PLACE.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd:115]:
```suggestion
							<xsd:documentation>A STOP PLACE.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_taxiPlace_version.xsd [hunk 2]
```diff
@@ -60,6 +60,22 @@ Rail transport, Roads and Road transport
...
+							<xsd:documentation>TAXI RANK.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
 	<xsd:element name="TaxiRank" abstract="false" substitutionGroup="StopPlace_">
 		<xsd:annotation>
 			<xsd:documentation>A place comprising one or more locations where taxis may stop to pick up or set down passengersA place comprising one or more locations where taxis may stop to pick up or set down passengers.  +v1.2.2
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_taxiPlace_version.xsd:81]:
```suggestion
			<xsd:documentation>A place comprising one or more locations where taxis may stop to pick up or set down passengers.  +v1.2.2
```

**Review by trurlurl — CHANGES_REQUESTED** [review]

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd (hunk 1 above)
Comment (skinkie) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd:115]: @ue71603 The comment regarding Vehicile Meeting Point, is this something we are still missing.

**Comment by ue71603** [comment]
@skinkie  no VehicleMeetingPoints have their own structure. So we are good, I think.

**Comment by ue71603** [comment]
The PR was no master so expanded it in master.

Von: Stefan de Konink ***@***.***>
Gesendet: Donnerstag, 25. Januar 2024 07:08
An: NeTEx-CEN/NeTEx ***@***.***>
Cc: Günter Matthias (IT-PTR-EXT-EXT2 - Extern) ***@***.***>; Mention ***@***.***>
Betreff: Re: [NeTEx-CEN/NeTEx] Revert new modes StopPlace_ abstract on stopPlacesInFrame_RelStructure (PR #513)


Then is the introduction of taxiRanks we want to do at master or at next? I would say this is a fix on master.

-

**Review by trurlurl — APPROVED** [review]

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_version.xsd]: no.... all set in different elements.

**Pull request merged**